### ISSUE TITLE
DRAFT watermark on spec document.

### ISSUE THREAD

**Issue by NeilMcKechnie** [openreferral/specification#94]
There is still a watermark of DRAFT even though the spec was approved a long time ago.

https://docs.google.com/document/d/1dkipRm_YL3K28oTSG-NFZRMHYiv10ZZJzJlakZglOyc/edit?pref=2&pli=1

**Comment by monfresh** [comment]
Great point. I wonder if it would be better to remove the link to the Google Doc and just keep the Markdown-formatted spec that's in this repo. That way, whoever is maintaining the spec doesn't have to worry about keeping multiple versions in sync.

**Comment by NeilMcKechnie** [comment]
Your call. I just think the DRAFT would be off- putting to someone new to Open Referral. I am not new and was wondering if I was looking at the right version, since I knew it had been finalized.

**Comment by greggish** [comment]
re: moving the spec from Google Docs to Github-only, I think that's
probably something we should do. However I haven't figured out the right
setup yet; will open another issue.

**Comment by devinbalkind** [comment]
I'd recommend continuing to maintain a Google Docs version so that people
can comment on specific parts of it using a familiar system. I doubt many
practitioners engaged in human services work will want to register and
learn Github to make comments.

In the Google Doc I'd say "For the most up to date stable draft of the
technical specification, go to the Github spec."

I don't think you need to point people from the Github to the Google Doc.
If they're already familiar with Github it's best to keep them interacting
there.

**Comment by greggish** [comment]
@devinbalkind see issue #96

**Comment by devinbalkind** [comment]
Thanks but I read that already and deliberately posted the comment on this
thread.

**Comment by timgdavies** [comment]
The docs at http://docs.openreferral.org/en/latest/reference/ include hypothes.is to allow for user-friendly comments without people needing to get into the GitHubs. 

I'll add documentation of how this can be used to the next version of the docs. 

@greggish Can you update the [google doc](https://docs.google.com/document/d/1dkipRm_YL3K28oTSG-NFZRMHYiv10ZZJzJlakZglOyc/edit?pref=2&pli=1) to indicate that it is now deprecated and people should work from the docs website? 

Once that is done, safe to close this issue.

**Comment by greggish** [comment]
Done! 

I may still have to find other mentions of the old Github / Docs URL in other documentation, but in the meantime, the HSDS v1.0 Google Doc is deprecated via the header and redirects to docs.openreferral.org
